src/main/java/com/kodilla/ecommercee/domain/Cart.java

```diff
@@ -9,11 +9,12 @@
 import javax.persistence.*;
 import java.util.List;
 
-@NoArgsConstructor
+@NoArgsConstructor(force = true)
 @AllArgsConstructor
```

Comment:
Why did you change this annotation for `@NoArgsConstructor`?

Reply:
The '@NotNull' annotation imposes restrictions on the fields of the class, which means that the no-argument constructor generated by the '@NoArgsConstructor' annotation will not initialize these fields. To avoid NullPointerException errors, you should add (force = true), which enforces the generation of a constructor for fields with the '@NotNull' annotation.
![NotNull](https://user-images.githubusercontent.com/113088417/232193687-4eb8b400-c792-4f4f-9e27-30658e7ccd5b.gif)

Reply:
Hmm, we didn't need to use it before. I think we should asked Andrzej to explain that.